.github/workflows/postman-collection-runner.yml

```diff
@@ -1,6 +1,8 @@
 name: Postman Collection API test
 
 on:
+  pull_request:
```

Comment:
I don't remember adding this. However, I would recommend keeping this with some extra modifications done to it. Like, it should only work for newly integrated connectors when the developer of that connector raises a PR for that. Thoughts?

I'll remove this if you think this will be useless.

Reply:
I don't think it's easy to filter and run a workflow on newly added files in a pull request. Moreover, we'll end up spamming the connectors each time a pull request is updated with new commits, which wouldn't be preferable. Let's only keep the schedule trigger for now. If absolutely required, consider adding a merge queue trigger.